tests/spec/fs.chmod.spec.js

```diff
@@ -0,0 +1,74 @@
+var Filer = require('../..');
+var util = require('../lib/test-utils.js');
+var expect = require('chai').expect;
+
+describe('fs.chmod, fs.fchmod', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should be functions', function() {
+    var fs = util.fs();
+    expect(typeof fs.chmod).to.equal('function');
+    expect(typeof fs.fchmod).to.equal('function');
+  });
+
+  it('should automatically set mode=755 for a directory', function(done) {
+    var fs = util.fs();
+
+    fs.mkdir('/dir', function(err) {
+      if(err) throw err;
+
+      fs.stat('/dir', function(err, stats) {
+        if(err) throw err;
+        expect(stats.mode & 0o755).to.equal(0o755);
```

Comment:
Typo here? `0o755` -> `0x755`?

Reply:
NM, it's supposed to be octal.

Reply:
Not sure why the automated tests are failing. Assuming these ran correctly on your machine.

Reply:
Oh, I see. Travis is using Node `0.10` to run our tests, and it doesn't understand octal numbers. We should switch to a newer version of Node. I'd suggest `8.11.3`, which is the current LTS build.

Reply:
I've updated the Travis config on `develop`. If you do a rebase on your PR it should build and test cleanly now.

Reply:
Good catch!  I've rebased to pick up the version change on Travis, and it's passing now.